cupy/cuda/memory.pyx

```diff
@@ -394,6 +412,51 @@ cdef class SingleDeviceMemoryPool:
             DeprecationWarning)
         self.free_all_blocks()
 
+    cpdef size_t realloc(self, size_t ptr, Py_ssize_t new_size):
```

Comment:
Is `new_size` actually always same with current size when called from realloc_all?

Reply:
Yes, that's right.
Now, there is no use case to change array size dynamically in cupy, so new_size is always the same as current size.
Why is there the argument of new_size? I just want to make the method general purpose.

Reply:
Thanks! I may not publish this method, and make non-general. I will think of this.